.travis.yml

```diff
@@ -5,7 +5,6 @@ cache:
         - $HOME/.composer/cache
 
 php:
-  - 5.3
```

Comment:
I see.

Just noticed PHP 5.3 is EOL http://php.net/supported-versions.php, I suppose now is a good time to drop 5.3 support.